src/APIFunctions/Event.js

```diff
@@ -1,18 +1,27 @@
 import axios from 'axios'
+// import {ApiResponse} from './ApiResponses'
+
+class ApiResponse {
+  constructor () {
+    this.error = false
+    this.responseData = null
+  }
+}
 
 /**
  * Retrieve all events.
  * @returns {Object[]} an array of all events
```

Comment:
Be sure to update the documentation of this comment, we don't return a boolean anymore

```suggestion
 * @returns {ApiResponse} Containing any error information related to the 
 * request or the list of events
```